pgxn/neon/neon--1.0.sql

```diff
@@ -22,6 +22,13 @@ AS 'MODULE_PATHNAME', 'backpressure_throttling_time'
 LANGUAGE C STRICT
 PARALLEL UNSAFE;
 
+CREATE FUNCTION num_cpus()
+RETURNS int
+AS 'MODULE_PATHNAME', 'num_cpus'
+LANGUAGE C STRICT
+PARALLEL UNSAFE
+VOLATILE;
```

Comment:
Where are we going to use it? I see that in `compute_ctl` you use Rust crate with the same name

Also keep in mind that we currently do not create `neon` extension in the computes

Reply:
Oh, I hoped that we were creating it. Then, it makes no-sense

The idea behind that was to share with customers from the very beginning to allow querying them how many CPUs their instances use

Reply:
To make it work one need to create it in the current db. We can mark `neon` as trusted (if we don't do it already), so users can create it in any database if needed. Maybe some other ACL tweaks on the func itself are also needed to make it usable by non-superusers

Reply:
Also, if we **were** creating the extension, then putting something into `neon--1.0.sql` won't affect any instance / db with already created extension. Usually people write upgrade SQL scripts (like `neon--1.0--1.1.sql`, IIRC) and change control file if they add new objects and expect that somebody already uses the extension

Reply:
I don't think that it's a good idea to allow such an installation for all users (for instance, views and functions related to `file_cache`). Maybe it's better to create an additional small extension and put it nearby

Reply:
moved into the newly created `neon_utils` extension